docs/source/pillars/data_management.md

```diff
@@ -81,23 +81,18 @@ The ability of the {ref}`TRE operator <infrastructure_roles>` to manage how and
     You should have a method of providing proof of deletion if challenged.
   - Recommended
 * - 3.1.11.
-  - You could keep backups of data and research environments, provided that this is permitted by law.
```

Comment:
@manics just noticed after merging that this was also removed - was this intentional?

Reply:
Oh I see, moved to Vulnerability Management: Resilience Processes